src/RouteService/RouteService.Domain/Entities/Ride.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace RouteService.Domain.Entities
+{
+    public class Ride
+    {
+        public int Id { get; set; }
+        public string RouteId { get; set; } = string.Empty;
+        public DateTime DepartureTime { get; set; }
+        public DateTime ArrivalTime { get; set; }
+        public string From { get; set; } = string.Empty;
+        public string To { get; set; } = string.Empty;
+        public int NumberOfSeats { get; set; }
```

Comment:
What does NumberOfSeats mean here? The total number of seats on this ride? Or total number of seats on the whole route?

Reply:
I use this number to produce booked seats